src/OpenLoco/src/Vehicles/Vehicle.cpp

```diff
@@ -686,6 +686,88 @@ namespace OpenLoco::Vehicles
         return AirportObjectFlags::acceptsHeavyPlanes;
     }
 
+    // 0x004AFFF3
+    VehicleBogie* VehicleBogie::flipCar()
+    {
+        Vehicle train(head);
+        VehicleBase* precedingCarBody = train.veh2;
```

Comment:
this doesn't have to be a body so should probably call it `precedingVehicleComponent`